pandas/core/frame.py

```diff
@@ -1127,6 +1127,49 @@ def from_dict(cls, data, orient='columns', dtype=None, columns=None):
 
         return cls(data, index=index, columns=columns, dtype=dtype)
 
+    def to_numpy(self):
+        """
+        Convert the DataFrame to a NumPy array.
+
+        .. versionadded:: 0.24.0
+
+        The dtype of the returned array will be the common NumPy
+        dtype of all types in the DataFrame. This may require copying
+        data and coercing values, which may be expensive.
```

Comment:
In the `DataFrame.values` docstring there is a longer note explaining this "lower-common-denominator". Maybe partly copy that here as well?